README.md

```diff
@@ -38,6 +38,13 @@ The project uses the [config](https://www.npmjs.com/package/config) npm package
 
 [(back to top)](#bsi-secvisogram-csaf-validator-service)
 
+## Using in production
+
+1. Install production dependencies using `npm ci --omit=dev` from within the `backend` folder
+2. Transfer the `backend` folder to your server or image
+3. Make sure to set the environment variable `NODE_ENV` to `production
+4. Configure the service using a `local-production.json` file in `backend/config`. All available parameters are outlined in `backend/config/development.json`. See [https://www.npmjs.com/package/config](https://www.npmjs.com/package/config) for more information on how to configure using different techniques such as environment variables.
```

Comment:
Shouldn't the `default.json` be referenced here? Do we recommend the use a reverse proxy (e.g. "The use of a reverse proxy is recommended.")? (If not, that's also okay - just checking; if so, why?)

Reply:
The usage of a separate configuration should be the preferred way. I added the recommendation of a reverse proxy.